FlowCrypt/src/main/java/com/flowcrypt/email/api/email/protocol/OpenStoreHelper.kt

```diff
@@ -145,7 +148,25 @@ class OpenStoreHelper {
               else -> session.getStore(JavaEmailConstants.PROTOCOL_IMAPS)
             }
 
-            store.connect(account.imapServer, account.username, account.password)
+            val password = if (account.useOAuth2) {
+              val accountManager = AccountManager.get(context)
+              val oauthAccount = accountManager.accounts.firstOrNull { it.name == account.email }
+              if (oauthAccount != null) {
+                val encryptedToken = accountManager.blockingGetAuthToken(oauthAccount,
+                    FlowcryptAccountAuthenticator.AUTH_TOKEN_TYPE_EMAIL, true)
+                if (encryptedToken.isNullOrEmpty()) {
+                  ""//need to think about
```

Comment:
Can consider to send ACRA report if you need to know how often does this happen

Reply:
Agree. I will add changes